sql/mob_skill_lists.sql

```diff
@@ -3950,7 +3950,13 @@ INSERT INTO `mob_skill_lists` VALUES ('Unfettered_Twitherym',2017,2952); -- blac
 
 INSERT INTO `mob_skill_lists` VALUES ('Unfettered_Twitherym_low_50',2018,2953); -- smouldering_swarm
 
--- Next available ID: 2019
+INSERT INTO `mob_skill_lists` VALUES ('Bomb_TOAU',2019,511); -- Berserk
```

Comment:
From your own capture linked, the skill ID for berserk used by this mob is ID 593
`[Actor: 17031401 (Big Bomb)] Berserk > Cat: 11 ID: 593 Anim: 879 Msg: 194`

Reply:
Interesting. I wasn't able to discern any new behaviors in berserk.

Reply:
There are many copies of TP moves. Probably a mere matter of having different power than other copies

Reply:
I can create a similar "dummy" file to the other bomb skills that were preexisting this PR. they were simply p1_hellstorm, p1_vulcanian_impact, etc.
That way the ID is reserved, and a separate file is made for further balancing if needed.

Reply:
Similarily, heat wave from big bomb was ID 595

Reply:
For now just correct the ID here in this skill list.

Reply:
Before I commit changes, I'm going to get another capture of Friar's Lantern behaviors + berserk ID to confirm all is well.
Will link my findings!